index.bs

```diff
@@ -742,6 +743,60 @@ self.onmessage = async function(e) {
 }
 </pre>
 
+# Implementation advice # {#implementation-advice}
+
+This section is informative.
+
+## Use with multiple consumers
+
+There are use cases where the programmer may desire that a single stream of frames
+is consumed by multiple consumers.
+
+Examples include the case where the result of a background blurring function should
+be both displayed in a self-view and encoded using a {{VideoEncoder}}.
+
+However, the standard tee() operation is problematic
+in this context:
+
+*   It defeats the backpressure mechanism that guards against excessive queueing
+*   It creates multiple links to the same buffers, meaning that the question of which
+    consumer gets to destroy() the buffer is a difficult one to address
+
+Therefore, the use of tee() with Streams containing media should only be done when
+fully understanding the implications. Instead, custom elements for splitting streams
+more appropriate to the use case should be used.
+
+For cases where both consumers are consuming unprocessed frames, and synchronization
+is not desired, instantianting multiple {{MediaStreamTrackProcessor}} objects is a robust solution.
+
+For cases where both consumers intend to convert the result of a processing step into a
+{{MediaStreamTrack}}
+using a {{MediaStreamTrackGenerator}}, attaching the resulting {{MediaStreamTrack}} to
+multiple sinks may be the most appropriate mechanism.
```

Comment:
It would be good to add an example to this one.
Is this when you want to do a single conversion (e.g., add funny hat), and send the result to two sinks (e.g., PC and element?)

Reply:
Added example in-line.